### DISCUSSION TITLE
Filter Table by comparing values from different comumns

### DISCUSSION THREAD

**Discussion by zaicef** [koalyptus/TableFilter#815] [Ideas]
Hello! I am wondering if the library supports filtering by column comparison.
eg. Display Table Records where  Column B is Greater than Column A.
Thank you!

**Comment by cjohnsonuk** [comment]
You'd have to create an extra column populated with a value indicating in
col a was bigger than col b then filter on that. It can be hidden and set
as a default filter if need be. Either in your app that builds the table of
using a .js on the page and create it within the browser

On Thu, 27 May 2021 at 22:31, zaicef ***@***.***> wrote:

> Hello! I am wondering if the library supports filtering by column
> comparison.
> eg. Display Table Records where Column B is Greater than Column A.
> Thank you!
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/koalyptus/TableFilter/discussions/815>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABOSJITLWFMAEXUWBRJM4F3TP22ZRANCNFSM45VEEPSA>
> .
>
-- 
Chris Johnson

**Reply by zaicef** [reply]
Thanks. I will follow your workaround!
